### PR TITLE
Fixes #7551: Some titles broken in the documentation

### PR DESCRIPTION
**Pull request by amousset**
http://www.rudder-project.org/redmine/issues/7551

### CODE REVIEW

**Comment by amousset** [comment]
The title level is limited to `=====`, moving "Understanding how Technique Editor works" to the same level as "Recommended solution: Technique Editor".

File: 4_advanced_usage/30_technique_creation.txt [hunk 1]
```diff
@@ -279,14 +279,14 @@ include::technique-st-sample.xml
 
 ----
 
-====== Define variables
+===== Define variables
```
Comment (amousset) [inline 4_advanced_usage/30_technique_creation.txt]: Fixing the title hierarchy here, opened another [issue](http://www.rudder-project.org/redmine/issues/7592) for the content.

File: 4_advanced_usage/30_technique_creation.txt [hunk 2]
```diff
@@ -68,7 +68,7 @@ package so you should never modify anything here: it will be lost at some point.
 * /var/rudder/configuration-repository/ncf
 ** This is were you add your own ncf Generic Methods and Techniques.
 Techniques created with the Technique Editor will be located here, and both Generic Methods
-and Techniques in that place will be accessible in the Technique Editor alongs what is 
+and Techniques in that place will be accessible in the Technique Editor along what is
```
Comment (jooooooon) [inline 4_advanced_usage/30_technique_creation.txt]: So... "alongs" is definitely wrong, I agree. But it doesn't really make sense anyhow. This should probably we "alongside".

**Comment by amousset** [comment]
PR updated.

**Comment by jooooooon** [comment]
Thanks! This is great.

**Pull request merged**